public/views/partials/admin_pages/list.liquid

```diff
@@ -1,7 +1,3 @@
----
-slug: admin/users
-layout: modules/admin/application
----
 {% liquid
   function current_user = 'modules/user/lib/queries/user/current'
   include 'modules/permission/lib/helpers/can_do_or_unauthorized', requester: current_user, do: 'users.manage'
```

Comment:
might be redundant as the Admin module already checks the permissions passed in `hook_admin_page`